.npmrc

```diff
@@ -1,2 +1 @@
-provenance=true
```

Comment:
What's the reason for disabling provenance and removing the ID token?

Reply:
id-token - I don't udnerstand it 😂 I tried to google it up quickly but I couldn't find what exactly it does for us here. 

provenance - I wasn't quite sure if this would work, doesn't it require some extra setup? I recall people complaining that Changesets doesn't support it yet. I haven't had a chance to look into it though. I just went for the simplest setup possible, reflecting the one that we have in other repos. But certainly if we can make this work - it would be great to make it work but perhaps in follow up PRs?

Reply:
Sure happy to do it in follow up PRs. It should "just work" in theory, with the id token set up. Can test it here and then if it works apply it elsewhere =)